client/src/App.jsx

```diff
@@ -9,10 +9,13 @@ import FavoritePage from '../src/pages/FavoritePage';
 import DetailedGasStationPage from '../src/pages/DetailedGasStationPage';
 
 function App() {
+  const [fuelValue, setFuelValue] = useState('e5');
   const [favoriteIDs, setFavoriteIDs] = useState(
     () => JSON.parse(localStorage.getItem('favoriteIDs')) ?? []
   );
 
+  const FuelInfo = fuelValue.charAt(0).toUpperCase(0) + fuelValue.slice(1);
```

Comment:
What happens if the first char is a whitespace (is that possible)?
More importantly, `.toUpperCase()` does not get an argument, i.e. the 0 is useless.

Reply:
No its not. The information is never entered from the outside, it comes from the radio buttons from the settings.jsx 😊👍

Reply:
I agree with Dennis ☺️